MaciASL/MaciASL-Info-Debug.plist

```diff
@@ -74,11 +74,11 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>${CURRENT_MARKETING_VERSION}</string>
```

Comment:
That's not right to split the plist files. Especially if one of them stops updating. I believe we should rather have an action to fill `Info.plist` entries if that is particularly needed.

Reply:
Oops, this commit was by accident while I was testing the automatic software update pop up. Should be fixed

Reply:
One of the ways would be to use `Build Phases`. You can add some shell code there before `Archiving`. For example, you can go with `defaults` or `PlistBuddy`.

Reply:
can we use `plutil` as well?

Reply:
Yeah

Reply:
Done. Can you check if I did it correctly?

Reply:
"File not found" hmm

Reply:
Fixed "Build" but Clang is complaining... Is there a check if "analyze" is running?